src/holoscan_cli/common/enum_types.py

```diff
@@ -37,10 +37,17 @@ class Arch(Enum):
 
 
 class Platform(Enum):
+    # User values - update SDK.Platforms
+    Jetson = "jetson"
+    IGX_iGPU = "igx-igpu"
+    IGX_dGPU = "igx-dgpu"
+    SBSA = "sbsa"
+    x86_64 = "x86_64"
+
+    # Internal use only
```

Comment:
May want to elaborate a bit more on the comment of "Internal use". Not sure if for backwards compatibility or else.